test/b2gedmntuples_cfg.py

```diff
@@ -768,7 +768,7 @@
 process.boostedJetUserDataAK8 = cms.EDProducer(
     'BoostedJetToolboxUserData',
     jetLabel  = cms.InputTag('jetUserDataAK8'),
-    puppiSDjetLabel = cms.InputTag('packedPatJetsAK8PFPuppiSoftDrop'),
+    puppiSDjetLabel = cms.InputTag('slimmedJetsAK8'),
```

Comment:
This is wrong because we are **mimic** the puppi jets from CHS as in miniAOD (line 521), we are not taking the values directly from miniAOD. They can be similar, but I think we are safer doing everything internally.
Same for your line 804.